### DISCUSSION TITLE
Is it safe to handle user roles (authorization) from public schema table or from authentication & authorization mechanism provided by postgres?

### DISCUSSION THREAD

**Discussion by roker15** [supabase/supabase#3237] [Questions] [answered]
Is it safe to handle user roles (authorization) from public schema user table or from authentication & authorization mechanism provided by postgres.

**Comment by awalias** [comment, marked as answer]
hey @roker15 thanks for you question

as this is related to [my response](https://github.com/supabase/supabase/discussions/3232#discussioncomment-1343550) to one of your [previous questions](https://github.com/supabase/supabase/discussions/3232) it would be most useful to add this as a comment response to that thread in order to better help future users find answers, and to avoid spamming the main discussion board
